### ISSUE TITLE
live httpstreaming, f4m version 2: pixel picture at load

### ISSUE THREAD

**Issue by phloxic** [flowplayer/flash#219]
Will try to convince the customer to run the stream at http://mp4aptest.media.hinet.net/verifyPlayer/HDSPlayer_3.2.18_1.aspx?MediaPath=http://61.219.3.107/live/pool/aver-nal/hd-hdsv2-pc/aver-nal.f4m again.
Was confirmed by @bbbo and me though.

**Comment by danrossi** [comment]
Yes you aren't going crazy I will look at it when I can something has changed in the player causing this ..

**Comment by danrossi** [comment]
The display is displaying before it has metadata to resize to.

**Comment by danrossi** [comment]
I will integrate this dvr clip setting too for those who have no dvr information but live stream is configured. hopefully it can be overridden after the parser.

**Comment by danrossi** [comment]
unable to replicate with either live or dvr on wowza. That stream is down also.

**Comment by danrossi** [comment]
Am able to replicate it with another one of their streams. it's because no width and height properties are set which is not standard. The plugin however has code to counteract this already and resets the clip dimensions to the player dimensions however it's quite possible not even a metadata even is being called.

**Comment by danrossi** [comment]
doh that was something I had implemented into the hls plugin because dimensions are not something that is enforced. lol. that seems to have done the job.

**Comment by danrossi** [comment]
https://dl.dropboxusercontent.com/u/3394987/flowplayer.httpstreaming-3.2.11.zip this will fix the problem.
